src/NServiceBus.AcceptanceTests/Core/PublishSubscribe/When_disabling_publishing.cs

```diff
@@ -52,12 +52,13 @@ public EndpointWithDisabledPublishing()
             {
                 var template = new DefaultServer();
                 template.TransportConfiguration = new ConfigureEndpointAcceptanceTestingTransport(false, true);
-                // use a persistence which doesn't support subscription persistence
-                template.PersistenceConfiguration = new ConfigureEndpointLearningPersistence(false);
 
                 EndpointSetup(template,
                     // DisablePublishing API is only available on the message-driven pub/sub transport settings.
-                    (c, _) => c.GetSettings().Set("NServiceBus.PublishSubscribe.EnablePublishing", false),
+                    (c, _) => {
+                        c.GetSettings().Set("NServiceBus.PublishSubscribe.EnablePublishing", false);
+                        c.UsePersistence<InMemoryPersistence, StorageType.Subscriptions>();
```

Comment:
This seems to contradict the old comment but I had to add this to get the test green?